examples/oneapi/cpp/source/example_util/output_helpers_graph.hpp

```diff
@@ -0,0 +1,69 @@
+/*******************************************************************************
+* Copyright 2020 Intel Corporation
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*     http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*******************************************************************************/
+
+#pragma once
+
+#include <iomanip>
+#include <iostream>
+#include <numeric>
+
+#include "oneapi/dal/table/common.hpp"
+#include "oneapi/dal/algo/jaccard/vertex_similarity_types.hpp"
+#include "oneapi/dal/table/homogen.hpp"
+
+void print_vertex_similarity_result(
+    const oneapi::dal::preview::jaccard::vertex_similarity_result &result) {
+    using namespace oneapi::dal;
+    const std::int64_t nonzero_coeff_count = result.get_nonzero_coeff_count();
+    const std::int64_t lines_count = 20;
+    const std::int64_t print_count =
+        (nonzero_coeff_count < lines_count) ? nonzero_coeff_count : lines_count;
+
+    std::vector<std::int64_t> print_range(print_count);
```

Comment:
`#include <vector>` at the head of the file? For now, I suppose this dependency goes from the graph component

Reply:
Added